src/Solver/time_stepping/ActorState.cpp

```diff
@@ -105,5 +105,10 @@ void DynamicRuptureScheduler::setLastFaultOutput(long steps) {
 bool DynamicRuptureScheduler::hasDynamicRuptureFaces() const {
   return numberOfDynamicRuptureFaces > 0;
 }
+
+
+bool DynamicRuptureScheduler::getIsFirstDynamicRuptureCluster() const {
```

Comment:
Maybe it is better to rename `getIsFirstDynamicRuptureCluster` to `isFirstDynamicRuptureCluster`